src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -199,7 +199,8 @@ def make_parameter_file(
         azimuth_looks: int,
         range_looks: int,
         dem_name: str = 'GLO_30',
-        dem_resolution: int = 30) -> None:
+        dem_resolution: int = 3,
+        apply_water_mask: bool = False) -> None:
```

Comment:
It looks like we've opted to not provide a default value for the parameters that get passed explicitly to our call to `make_parameter_file`, so let's remain consistent with that. (And we'll probably want to move `apply_water_mask` up to right after `range_looks`.)
```suggestion
        apply_water_mask: bool) -> None:
```